src/SaveObject.ts

```diff
@@ -807,7 +890,7 @@ async function loadGame(saveData: SaveData): Promise<boolean> {
   }
   if (Object.hasOwn(saveObj, "LastExportBonus")) {
     try {
-      ExportBonus.setLastExportBonus(JSON.parse(saveObj.LastExportBonus));
+      ExportBonus.setLastExportBonus(JSON.parse(saveObj.LastExportBonus) as number);
```

Comment:
We should either leave these as unfixed, or actually fix them properly here in the load code IMO. In this case that would look something like throwing an error if the bonus is not a number, and letting the catch block handle setting a reasonable default.